build.gradle

```
@@ -499,6 +504,17 @@ startScripts {
   doLast {
     unixScript.text = unixScript.text.replace('BESU_HOME', '\$APP_HOME')
     windowsScript.text = windowsScript.text.replace('BESU_HOME', '%~dp0..')
+    String agentFileName = configurations.javaAgent.singleFile.toPath().getFileName()
+    String unixRegex = $/exec "$$JAVACMD" /$
+    String forwardSlash = "/"
+    String unixReplacement = $/if [ -n "$$TRACING" ];then
+ TRACING_AGENT="-javaagent:$$APP_HOME/agent${forwardSlash}${agentFileName}"
+fi
+exec "$$JAVACMD" $$TRACING_AGENT /$
+    unixScript.text = unixScript.text.replace(unixRegex, unixReplacement)
+    String windowsRegex = $/"%JAVA_EXE%" %DEFAULT_JVM_OPTS%/$
+    String windowsReplacement = $/if Defined TRACING (TRACING_AGENT="-javaagent:"%APP_HOME%\agent\/$ + agentFileName + '")\r\n"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %TRACING_AGENT%'
+    windowsScript.text = windowsScript.text.replace(windowsRegex, windowsReplacement)
```

Mostly style and doc nits.  We use `def` more than actual types in the build scripts, especially for basic types.

```suggestion
    // OpenTelemetry Wiring for unix scripts
    def agentFileName = configurations.javaAgent.singleFile.toPath().getFileName()
    def unixRegex = $/exec "$$JAVACMD" /$
    def forwardSlash = "/"
    def unixReplacement = $/if [ -n "$$TRACING" ];then
 TRACING_AGENT="-javaagent:$$APP_HOME/agent${forwardSlash}${agentFileName}"
fi
exec "$$JAVACMD" $$TRACING_AGENT /$
    unixScript.text = unixScript.text.replace(unixRegex, unixReplacement)
    
    // OpenTelemetry Wiring for windows scripts
    def windowsRegex = $/"%JAVA_EXE%" %DEFAULT_JVM_OPTS%/$
    def windowsReplacement = $/if Defined TRACING (TRACING_AGENT="-javaagent:"%APP_HOME%\agent\/$ + agentFileName + '")\r\n"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %TRACING_AGENT%'
    windowsScript.text = windowsScript.text.replace(windowsRegex, windowsReplacement)
```